Extension/src/LanguageServer/configurations.ts

```diff
@@ -386,6 +391,70 @@ export class CppProperties {
         }
     }
 
+    public nodeAddonIncludesFound(): number {
+        return this.nodeAddonIncludes.length;
+    }
+
+    private async readNodeAddonIncludeLocations(rootPath: string): Promise<void> {
+        let error: Error | undefined;
+        let pdjFound: boolean = false;
+        const package_json: any = await fs.promises.readFile(path.join(rootPath, "package.json"), "utf8")
+            .then(pdj => {pdjFound = true; return JSON.parse(pdj); })
+            .catch(e => (error = e));
+
+        const pathToNode: string = which.sync("node");
+        const nodeAddonMap: { [dependency: string]: string } = {
+            "nan": `${pathToNode} --no-warnings -e "require('nan')"`,
```

Comment:
${pathToNode} needs to be `"${pathToNode}"` in order to handle spaces in the path on Windows (I haven't checked Linux).

Reply:
just pushed. it would need them on linux too (though most people don't use spaces in paths). i've been a bit sloppy.

Reply:
would you like me to resolve these when i think i have or would you like to?

Reply:
Yeah, you can resolve them.